modules/testing/js/src/main/scala/JsInstances.scala

```diff
@@ -84,7 +84,7 @@ trait JsInstances {
   private[testing] def itemCollectionGen: Gen[ItemCollection] =
     (
       Gen.const("FeatureCollection"),
-      Gen.const(StacVersion.unsafeFrom("0.9.0")),
+      Gen.const(StacVersion.unsafeFrom("1.0.0-beta.2")),
```

Comment:
uff I guess that should be `ValidStacVersion.stacVersions.head`

Reply:
:yoda:

Reply:
Ha refined doesn't know that the strings in `stacVersions` satisfy the `ValidStacVersion` predicate 🤦‍♂️